docs/src/man/getting_started.md

```diff
@@ -95,8 +95,8 @@ ncols = size(df, 2)
 We can also look at small subsets of the data in a couple of different ways:
 
 ```julia
-head(df)
-tail(df)
+DataFrames.head(df)
```

Comment:
It doesn't seem that `head` is exported, so this is the only way this line works